app/controllers/admin/programmes_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Admin
+  class ProgrammesController < Admin::ApplicationController
+    def index
+      @programmes = [
+        Programme.cs_accelerator,
+        Programme.primary_certificate,
+        Programme.secondary_certificate,
+        Programme.i_belong,
+        Programme.a_level
+      ].compact.sort_by(&:title)
```

Comment:
This raised an eyebrow at first, but I then realised this is just 5 records being sorted in memory, not 5 sets of records.